R/get_14cpalaeolithic.R

```diff
@@ -0,0 +1,60 @@
+#' @rdname db_getter_backend
+#' @export
+get_14cpalaeolithic <- function(db_url = get_db_url("14cpalaeolithic")) {
+
+  check_connection_to_url(db_url)
+
+  # db_url <-"https://ees.kuleuven.be/geography/projects/14c-palaeolithic/radiocarbon-palaeolithic-europe-database-v26-extract.xlsx"
+
+
+  # download data
+  temp <- tempfile(fileext=".xlsx")
+
+  utils::download.file(url = db_url, destfile = temp, mode="wb")
+
+  # Read each file and write it to csv
+  lapply(temp, function(f) {
```

Comment:
Why this extra step of reading xlsx, writing to csv and reading again from csv?

Reply:
Because I was running into errors using the .xlsx-file